spec/Pim/Bundle/DataGeneratorBundle/Generator/AttributeOptionGeneratorSpec.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace spec\Pim\Bundle\DataGeneratorBundle\Generator;
+
+use PhpSpec\ObjectBehavior;
+use Pim\Bundle\CatalogBundle\Repository\LocaleRepositoryInterface;
+use Prophecy\Argument;
+
+class AttributeOptionGeneratorSpec extends ObjectBehavior
```

Comment:
DataGeneratorBundle is going to be rewrite in a big way soon. The main generator code will be reused, but in another form .